extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -132,6 +132,14 @@ export class AmpAutocomplete extends AMP.BaseElement {
      */
     this.activeElement_ = null;
 
+    /**
+     * The element id if present or random number.
+     * @private {number|string}
+     */
+    this.prefix_ = element.id
+      ? element.id
+      : Math.floor(Math.random() * Math.floor(100));
```

Comment:
There were a couple of post-merge feedback comments in the original PR (#18534) that I think we should add to both here and amp-accordion

- [ ] `Math.floor(100)` can be simplified to `100` https://github.com/ampproject/amphtml/pull/18534#discussion_r223865610
- [ ] should be `this.prefix_` in amp accordion https://github.com/ampproject/amphtml/pull/18534#discussion_r223865690

Would you mind doing that as part of this PR?